Dockerfile

```diff
@@ -8,15 +8,18 @@ RUN go mod download
 COPY . ./
 COPY .docker/entrypoint.sh start.sh
 
-
 RUN CGO_ENABLED=0 GOOS=linux go build -o /ptra
 
 # Run the tests in the container
 FROM build-stage AS run-test-stage
 RUN go test -v ./...
 
 # Deploy the application binary into a lean image
-FROM alpine AS build-release-stage
+FROM debian AS build-release-stage
```

Comment:
We could use `debian:stable-slim` to reduce the final image size. I'll give it a try.